PopPUNK/__main__.py

```diff
@@ -166,6 +176,15 @@ def get_options():
                                               '[default = False]', default=False, action='store_true')
     queryingGroup.add_argument('--accessory-only', help='Use an accessory-distance only model for assigning queries '
                                               '[default = False]', default=False, action='store_true')
+    queryingGroup.add_argument('--assign-lineage', help='Assign to lineages rather than strains - can specify '
+                                              'lineages other than those in the reference database using "--existing-scheme"'
+                                              '[default = False]', default=False, action='store_true')
+
+    # lineage clustering within strains
+    lineagesGroup = parser.add_argument_group('Lineage analysis options')
+    lineagesGroup.add_argument('--R',help='Comma separated list of ranks used in lineage clustering [default = 1]', type = str, default = "1")
```

Comment:
I feel like this could be better, but am not immediately sure how

Reply:
Comma separated is fine, perhaps just a more informative default (e.g. 1,2,5)

Reply:
Changed defaults in effb8909e329c8ac57e1bc9aea09be1088f7021d.